src/recipe/Baked.Recipe.Service/Communication/Response.cs

```diff
@@ -1,12 +1,18 @@
 using Newtonsoft.Json;
+using System.Net;
 
 namespace Baked.Communication;
 
 public record Response(
+    HttpStatusCode StatusCode,
```

Comment:
this is abstraction, shouldn't be using `HttpStatusCode`. create another enum to represent success (2xx), handled (4xx), and unhandled (5xx)